### DISCUSSION TITLE
Pickit code for replenish life?

### DISCUSSION THREAD

**Discussion by danmm55** [blizzhackers/kolbot#330] [Q&A about d2bot-with-kolbot]
what is the pickit code for replenish life?

**Comment by theBGuy** [comment]
`NTIPAliasStat["hpregen"] = 74;`

**Comment by danmm55** [comment]
thx a lot ive been trying to find that info for years lol. how can u find info for other stats? like cast skill lvl when struck? or items that roll defense? like defense+10-20 (varies)

**Comment by theBGuy** [comment]
You didn't try very hard then. It took me all of 30seconds to check
https://github.com/blizzhackers/kolbot/blob/master/d2bs/kolbot/libs/NTItemAlias.dbl
then confirm in game that the stat matched the one I thought it was.
Ctc skills don't have set stats in NTItemAlias meaning there isn't a way to look for them using pickit. For more references you can also look at this whole pickit repo
https://github.com/blizzhackers/pickits
